docs/en/latest/plugins/google-cloud-logging.md

```diff
@@ -0,0 +1,156 @@
+| auth_config             | Semi-optional |                                                                                                                                                                                                   | one of `auth_config` or `auth_file` must be configured                                                                                                                           |
+| auth_config.private_key | Required      |                                                                                                                                                                                                   | the private key parameters of the Google service account                                                                                                                         |
+| auth_config.project_id  | Required      |                                                                                                                                                                                                   | the project id parameters of the Google service account                                                                                                                          |
+| auth_config.token_uri   | Optional      | https://oauth2.googleapis.com/token                                                                                                                                                               | the token uri parameters of the Google service account                                                                                                                           |
+| auth_config.entries_uri | Optional      | https://logging.googleapis.com/v2/entries:write                                                                                                                                                   | google cloud logging service API                                                                                                                                                       |
+| auth_config.scopes      | Optional      | ["https://www.googleapis.com/auth/logging.read","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/logging.admin","https://www.googleapis.com/auth/cloud-platform"] | the access scopes parameters of the Google service account, refer to: [OAuth 2.0 Scopes for Google APIs](https://developers.google.com/identity/protocols/oauth2/scopes#logging) |
+| auth_config.ssl_verify  | Optional      | true                                                                                                                                                                                              | enable `SSL` verification, when set to `true`, the server certificate will be verified according to the CA certificate specified by the `ssl.lua_ssl_trusted_certificate` directive in the APISIX main configuration file.|
```

Comment:
Should we remove the `lua_ssl_trusted_certificate` stuff? It is used to trust self-signed certificates.

Reply:
done